src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2390,6 +2403,444 @@ def remove_triggered_webjob(cmd, resource_group_name, name, webjob_name, slot=No
+    for x in result:
+        # removes GUIDs from name and id
+        longName = x.name
+        if '_' in longName:
+            usIndex = longName.index('_')
+            shortName = longName[usIndex + 1:]
+        else:
+            shortName = longName
+        v_id = x.id
+        lastSlash = v_id.rindex('/')
+        shortId = v_id[:lastSlash] + '/' + shortName
+        # extracts desired fields
+        certThumbprint = x.cert_thumbprint
+        location = x.additional_properties["location"]
+        v_type = x.type
+        vnet_resource_id = x.vnet_resource_id
+        id_strings = v_id.split('/')
+        resourceGroup = id_strings[4]
+        routes = x.routes
+
+        vnet_mod = {"certThumbprint": certThumbprint,
+                    "id": shortId,
+                    "location": location,
+                    "name": shortName,
+                    "resourceGroup": resourceGroup,
+                    "routes": routes,
+                    "type": v_type,
+                    "vnetResourceId": vnet_resource_id}
+        mod_list.append(vnet_mod)
+
+    return mod_list
+
+
+def webapp_add_vnet_int(cmd, name, resource_group, vnet, subnet, slot=None):
+    webapp = _generic_site_operation(cmd.cli_ctx, resource_group, name, 'get', slot)
+    if webapp.kind == "app":
+        return add_vnet_int(cmd, name, resource_group, vnet, subnet, slot)
+    return logger.warning("not a webapp")
+
+
+def functionapp_add_vnet_int(cmd, name, resource_group, vnet, subnet, slot=None):
+    webapp = _generic_site_operation(cmd.cli_ctx, resource_group, name, 'get', slot)
+    if webapp.kind == "functionapp":
+        return add_vnet_int(cmd, name, resource_group, vnet, subnet, slot)
+    return logger.warning("not a functionapp")
+
+
+def add_vnet_int(cmd, name, resource_group, vnet, subnet, slot):
+    client = web_client_factory(cmd.cli_ctx)
+    vnet_client = network_client_factory(cmd.cli_ctx)
+
+    list_all_vnets = vnet_client.virtual_networks.list_all()
+
+    vnet_id = ''
+    for v in list_all_vnets:
+        if v.name == vnet:
+            vnet_id = v.id
+
+    # parsing the arm uri in order to extract vnet_name and vnet_resource_group
+    vnet_id_strings = vnet_id.split('/')
+
+    vnet_resource_group = ''
+    i = 0
+    for z in vnet_id_strings:
+        if z == "resourceGroups":
+            vnet_resource_group = vnet_id_strings[i + 1]
+        i = i + 1
+
+    vnet_info = vnet_client.virtual_networks.get(vnet_resource_group, vnet)
+
+    if slot is None:
+        swift_connection_info = client.web_apps.get_swift_virtual_network_connection(resource_group, name)
+    else:
+        swift_connection_info = client.web_apps.get_swift_virtual_network_connection_slot(resource_group, name, slot)
+
+    # check to see if the connection would be supported
+    if swift_connection_info.swift_supported is not True:
+        logger.warning("""Your app must be in an Azure App Service deployment that is
```

Comment:
Don't completely understand this message here. Is this something Christina shared? Also if the connection is not supported should be stop the execution here so have a return here?

Reply:
Never mind you have a return can we just to return logger.warning(...)